lib/db/leveldb_dbinstance.go

```diff
@@ -88,13 +89,21 @@ func (db *Instance) UpdateSchema() {
 	miscDB := NewNamespacedKV(db, string(KeyTypeMiscData))
 	prevVersion, _ := miscDB.Int64("dbVersion")
 
+	if prevVersion == dbVersion {
```

Comment:
`>=` so we avoid downgrades? (Until we for some reason need downgrades and handle them correctly.)